src/modules/firebase/models/Collections.ts

```diff
@@ -0,0 +1,3 @@
+export enum Collections {
```

Comment:
Prefer string unions when key and value are identical. They have the same or better code completion compared to enums but do not encapsulate the value unnecessarily.

`type Collections = 'users' | 'something-else'`